web/spec/supabase.yml

```diff
@@ -151,7 +151,17 @@ pages:
           const options = { }
           const supabase = createClient("https://xyzcompany.supabase.co", "public-anon-key", options)
           ```
+      - name: With additional parameters - access non-public schemas
+        js: |
+          ```js
+          import { createClient } from '@supabase/supabase-js'
+
+          const options = {schema: "otherschema"}
+          const supabase = createClient("https://xyzcompany.supabase.co", "public-anon-key", options)
+          ```
+          Note: each client connection can only access a single schema, so the code above can access the `otherschema` schema but cannot access the `public` schema.
 
+          TODO:  Descibe how to whitelist the schema from the Supabase dashboard, which is required to obtain access to the schema from the client.
```

Comment:
I just added this in staging, so it should be live by tomorrow 👍

Reply:
Where can I see how to whitelist items in the dashboard so I can add this to the documentation?